### PR TITLE
aggs: Fixes boolean values bug

### PR DESCRIPTION
**Pull request by jrcastro2**
* Adds check for boolean values on aggs

Fixes boolean filter when selected

closes https://github.com/inveniosoftware/react-searchkit/issues/56

### CODE REVIEW

File: src/lib/components/BucketAggregation/BucketAggregationValues.js [hunk 1]
```diff
@@ -67,16 +67,15 @@ class BucketAggregationValues extends Component {
     const valuesCmp = buckets.map((bucket) => {
       const isSelected = this._isSelected(
         this.aggName,
-        bucket.key,
+        bucket.key_as_string ? bucket.key_as_string : bucket.key,
```
Comment (kprzerwa) [inline src/lib/components/BucketAggregation/BucketAggregationValues.js:70]: yessss
@ntarocco why did we reject this change in the past, do you remember?

**Review by kprzerwa — APPROVED** [review]

File: src/lib/components/BucketAggregation/BucketAggregationValues.js (hunk 1 above)
Comment (ntarocco) [reply src/lib/components/BucketAggregation/BucketAggregationValues.js:70]: Not sure, I can't remember... I would also add the same change to the label below maybe?

```
const key_field = bucket.key_as_string ? bucket.key_as_string : bucket.key
const label = bucket.label ? bucket.label : `${bucket["key_field"]} (${bucket.doc_count})`
```

Comment (ntarocco) [inline src/lib/components/BucketAggregation/BucketAggregationValues.js:70]:
```suggestion
const key_field = bucket.key_as_string ? bucket.key_as_string : bucket.key
const isSelected = this._isSelected(this.aggName, bucket["key_field"], selectedFilters)
```

Comment (zzacharo) [reply src/lib/components/BucketAggregation/BucketAggregationValues.js:70]: I think you meant to write `bucket[key_field]` without quotes, right?

Comment (zzacharo) [reply src/lib/components/BucketAggregation/BucketAggregationValues.js]: @ntarocco @kprzerwa I had already an issue with this https://github.com/inveniosoftware/react-searchkit/issues/56 :)

**Review by ntarocco — APPROVED** [review]

**Review by kprzerwa — APPROVED** [review]

**Pull request merged**